src/app/sidebar/sidebar.component.ts

```diff
@@ -39,6 +40,15 @@ import { Subscription } from 'rxjs/Subscription';
         height: calc(100vh - var(--header-height) - 20px);
       }
     }
+
+    .content::-webkit-scrollbar {
```

Comment:
When is removing the scrollbar beneficial?

Reply:
Actually only on desktop. Even if it's not removed, it's good to have it visually less prominent.